src/steps/success.ts

```diff
@@ -1,11 +1,19 @@
-import type { Context } from 'semantic-release';
+import type { BranchObject, Context } from 'semantic-release';
 import { execSync } from 'node:child_process';
 
 import prepareTags from './prepareTags';
 import type { PluginConfigType } from './types';
 
+function isPrerelease(branchObject: BranchObject) {
+  return branchObject.prerelease !== undefined && branchObject.prerelease !== false
+}
```

Comment:
Would you be fine with using an error function instead to keep it unified across the project? :D  
We we can directly:
```suggestion
const isPrerelease = (branchObject: BranchObject) =>
  branchObject.prerelease !== undefined && branchObject.prerelease !== false
```